elide-contrib/elide-dynamic-config-helpers/pom.xml

```diff
@@ -42,20 +42,26 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <hjson.version>3.0.0</hjson.version>
-        <everit.version>1.0.0</everit.version>
         <jackson.version>2.10.2</jackson.version>
         <commons-lang3.version>3.0</commons-lang3.version>
+        <commons-io.version>2.6</commons-io.version>
```

Comment:
Any version that is in the parent pom shouldn't be redefined here - but rather inherited from the parent.    Take a look and remove all that fit that description.  That should include lombok, slf4j, and others.